pandas/_libs/tslibs/ccalendar.pxd

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+
+from cython cimport Py_ssize_t
+
+from numpy cimport int64_t, int32_t
+
+
```

Comment:
can we call this ``calendar.pxd`` ?

Reply:
We should avoid overlap with stdlib names.  The thought behind `ccalendar` is by analogy with  `chardet` --> `cchardet`

Reply:
i don't think that's actually a problem, I guess ccalendar is ok.